backend/homework/1/data/src/main/java/org/handson/StudentRepo.java

```diff
@@ -0,0 +1,84 @@
+package org.handson;
+
+public class StudentRepo {
+    static Student[] studentObjects;
+    static int n = 0;
+
+    public void addStudent(Student student) {
+      try
+      {
+        Student newarr[] = new Student[n + 1];
+        for (int i = 0; i < n; i++) {
+            newarr[i] = studentObjects[i];
+        }
+        newarr[n] = student;
+        studentObjects = newarr;
+        CustomLogging.getLoggerCustom().debug("Student added!");
+        n++;
+
+    }
+      catch (Exception e) {
+          CustomLogging.getLoggerCustom().error("Unable to add student!");
+
+      }
+      }
+
+    public Student getStudent(int id) {
+        for (var student : studentObjects) {
+            if ((student.getID()) == (id)) {
+                CustomLogging.getLoggerCustom().debug("Student info retrieved for provided ID");
+                return student;
+            }
+        }
+        CustomLogging.getLoggerCustom().error("Unable to get student info retrieved for provided ID");
+
+        return null;
```

Comment:
If there is an exception then there is no need to return a null the current action is already finished and handled.